scipy/spatial/_spherical_voronoi.py

```diff
@@ -214,6 +223,8 @@ def __init__(self, points, radius=None, center=None):
             self.radius = radius
         else:
             self.radius = 1
+        if pdist(self.points).max() > (2. * self.radius):
+            raise ValueError("Radius inconsistent with generators.")
```

Comment:
Floating point error is certainly a concern here (and for the other check as well, I think). Presumably there is a floating point sensitivity limit beyond which we can't be certain that the user input and provided radius are consistent. What do you mean by:

> compute the distance of the points?

You are correct that the sphere is assumed to be centered at the origin if the user does not specify a value for the `center` argument.